ui/app/adt/app.js

```diff
@@ -93,6 +93,12 @@ angular.module('adt').config(['$stateProvider', '$httpProvider', '$urlRouterProv
         });
 
         $bahmniTranslateProvider.init({app: 'adt', shouldMerge: true});
-    }]).run(['$window', function ($window) {
-        moment.locale($window.localStorage["NG_TRANSLATE_LANG_KEY"] || "en");
+    }]).run(['$rootScope', '$window', function ($rootScope, $window) {
+        // moment.locale($window.localStorage["NG_TRANSLATE_LANG_KEY"] || "en");
+        $rootScope.languageUser = $window.localStorage["NG_TRANSLATE_LANG_KEY"] || "en";
+        $rootScope.userLanguageDirRtl = false;
+        if ($window.localStorage["NG_TRANSLATE_LANG_KEY"] === 'ar') {
```

Comment:
Seems like this code is written to enable RTL only for Arabic locale. What about other Locales? Isn't there a way to directly check if a locale is RTL or not, instead of hardcoding RTL for specific locales? 

Also, multiple modules will want this check of detecting if current locale is RTL or not.. shouldn't that code be in one place? I see this code is repeated in adt/app.js, aapointment/app.js, bedmanagement/app.js, etc.